CHANGELOG

```diff
@@ -117,6 +117,11 @@ Linear and nonlinear solvers
 
 New and updated examples and miniapps
 -------------------------------------
+
+- Added a new block solver in miniapp/solvers for the Darcy problem.
+  The new solver is based on a Bramble-Pasciak preconditioning. User can
+  use and implement their own preconditioner for the mass matrix.
+
```

Comment:
Please move this to the mfem-4.6.1 section 😁

Reply:
Ok!